docs/en_us/platform_api/source/links.rst

```diff
@@ -1 +1,7 @@
-.. _OAuth 2.0 Standard: https://tools.ietf.org/html/draft-ietf-oauth-v2-31
\ No newline at end of file
+.. _OAuth 2.0 Standard: https://tools.ietf.org/html/draft-ietf-oauth-v2-31
+
+.. _edX platform documentation: http://docs.edx.org/
+
+.. _https://courses.edx.org/api/courses/v1/courses/: https://courses.edx.org/api/courses/v1/courses/
+
```

Comment:
I chose to show the URL in the text because I wanted to highlight the REST path in it. So I'm using the URL as the key for this link.